misk-jobqueue-testing/src/test/kotlin/misk/jobqueue/FakeTransactionalJobQueueTest.kt

```diff
@@ -215,6 +240,17 @@ private class Starport @Inject private constructor(moshi: Moshi) : JobHandler {
   }
 }
 
+private class Crasher @Inject private constructor(val tokenGenerator: TokenGenerator) : JobHandler {
+
+  override fun handleJob(job: Job) {
+    // Tokens are generated once per test run, starting from 1.
+    if (tokenGenerator.generate().endsWith("1")) {
+      throw UnitException()
```

Comment:
Just curious: Is there a significance to `UnitException`, or could it be any `Exception` class?

Reply:
Could be any checked exception, this one is keeping with the starcraft theme of this test class.